packages/cmf-cqrs/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "@talend/react-cmf plugin for CQRS backend architecture",
   "main": "lib/index.js",
   "scripts": {
-    "prepublish": "npm run build && npm run test",
+    "prepublish": "npm run build",
```

Comment:
We run tests separately, so having that usually end up running the tests twice, and slowing the build process